test

```diff
@@ -63,3 +63,4 @@ test coroutines-bluebird.js
 test await.compiled.js
 test tasks.js
 test lazy-either.js
+test bluebird-promisell.js
```

Comment:
Let's group this with the names of the other Promise-related files. Also, include a link to the file in the readme. :)